### PR TITLE
update application profile and readme

### PR DESCRIPTION
**Pull request by jacobthill**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ For data providers, start by reading our [Information for Data Providers](provid
 
 At present, this diagram represents how data travels from being queued up for inclusion to internal representation in the DLME application's database and metadata included the discovery index:
 
-![overview diagram](https://docs.google.com/drawings/d/e/2PACX-1vTBFJJgiPqs58fNWC-lTBdw5wKNN0-OgLBu7EUoJcfyDXFu6VTKkhxNUKcNSX4f1Mf_mHHI2zH_ezZj/pub?w=960&h=720)
-[Link to diagram in Google Drawings](https://docs.google.com/drawings/d/1jEspB9tO6-_LyiN-q0jQwfEPtiaztgHzL6CgRKXiyBk/edit)
```
Comment (jcoyne) [inline docs/README.md]: Please restore the editable link.

File: docs/application_profile.md [hunk 2]
```diff
@@ -40,9 +40,14 @@ The DLME Prototype Application Profile a subset of the [Europeana Data Model](ht
 @prefix svcs: <http://rdfs.org/sioc/services#> .
 ```
 
-### `DLME lnternal Fields`
+### `DLME Internal Fields`
 | Solr field name  | Property          | Expectation | Cardinality | Value type  | Definition | Notes                      |
 | ---------------- | ----------------- | ----------- | ----------- | ----------- | ---------- | -------------------------- |
+| agg_data_provider_collection  | | M           | 1...1       | literal     | The collection of which the resource is a part. | |
+| agg_data_provider_country  | | M           | 1...n       | literal     | The country of the organization or entity that supplies data through a provider. | |
```
Comment (anarchivist) [inline docs/application_profile.md]: All the country-related fields should be the property `edm:country`

File: docs/application_profile.md [hunk 3]
```diff
@@ -40,9 +40,14 @@ The DLME Prototype Application Profile a subset of the [Europeana Data Model](ht
 @prefix svcs: <http://rdfs.org/sioc/services#> .
 ```
 
-### `DLME lnternal Fields`
+### `DLME Internal Fields`
 | Solr field name  | Property          | Expectation | Cardinality | Value type  | Definition | Notes                      |
 | ---------------- | ----------------- | ----------- | ----------- | ----------- | ---------- | -------------------------- |
+| agg_data_provider_collection  | | M           | 1...1       | literal     | The collection of which the resource is a part. | |
```
Comment (anarchivist) [inline docs/application_profile.md]: This should be `edm:datasetName`.

**Comment by jacobthill** [comment]
@jcoyne @anarchivist I made all the requested changes.

**Pull request merged**